runners/flink/src/main/java/org/apache/beam/runners/flink/FlinkPipelineOptions.java

```diff
@@ -127,4 +127,15 @@
   @Default.Boolean(false)
   Boolean getRetainExternalizedCheckpointsOnCancellation();
   void setRetainExternalizedCheckpointsOnCancellation(Boolean retainOnCancellation);
+
+  @Description("Max elements size of Bundle trigger.")
```

Comment:
These could be a bit more descriptive, for example: `The maximum number of elements in a bundle.`